src/transformation-components/CombineCases.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useCallback, ReactElement } from "react";
+import { getContextAndDataSet } from "../utils/codapPhone";
+import { useInput } from "../utils/hooks";
+import { combineCases } from "../transformations/combineCases";
+import { DataSet } from "../transformations/types";
+import { ContextSelector, TransformationSubmitButtons } from "../ui-components";
+import { applyNewDataSet, ctxtTitle, addUpdateListener } from "./util";
+import { TransformationProps } from "./types";
+import TransformationSaveButton from "../ui-components/TransformationSaveButton";
+
+// eslint-disable-next-line
+export interface CombineCasesSaveData {}
```

Comment:
I think we could do 
```suggestion
export type CombineCasesSaveData = Record<string, never>;
```
which is the canonical way to represent an empty object. See: https://github.com/typescript-eslint/typescript-eslint/issues/2063#issuecomment-675156492. Then we can get rid of `eslint-disable-next-line`.

Reply:
Unfortunately this seems to cause a type error and I'm not sure why:

![Screenshot from 2021-06-08 10-56-28](https://user-images.githubusercontent.com/13399527/121208694-41149300-c848-11eb-9e8d-3678cbb1a50f.png)

Reply:
Huh, why is this happening in Compare...

Reply:
To be clear it happens in a lot of other transformations too once I change a few to use the `Record<string,never>`.

Reply:
I see. The type of `inputAttribute1` is `string | null`, but in the save data it's only allowed to be `string`.